docker-jans-all-in-one/app/jans_aio/monitor.py

```
@@ -18,7 +18,7 @@ def watch_jans_auth():
     try:
         host = os.environ.get("CN_AUTH_JETTY_HOST", "127.0.0.1")  # nosec: B104
         port = os.environ.get("CN_AUTH_JETTY_PORT", "8081")
-        req = requests.get(f"http://{host}:{port}/jans-auth/sys/health-check")
+        req = requests.get(f"http://{host}:{port}/sys/health-check")
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)